crates/holochain/Cargo.toml

```diff
@@ -188,7 +189,7 @@ path = "src/bin/holochain/main.rs"
 workspace = true
 
 [features]
-default = ["sqlite-encrypted", "tx2", "tx5", "metrics_influxive"]
+default = ["chc", "sqlite-encrypted", "tx2", "tx5", "metrics_influxive"]
```

Comment:
`cd crates/holochain && cargo build --no-default-features` doesn't compile

Do we want to make this a default feature? The majority of individual users won't use it I think?

Reply:
Well, it's gated behind an optional config anyway. And if anyone needs to reach for CHC, it would be handy for it to be compiled in. If anyone needs a smaller binary they could build without it.

I agree that it should compile without the feature though!

Reply:
It does now